javalib/src/main/scala/java/util/Collections.scala

```diff
@@ -249,14 +249,14 @@ object Collections {
     min(coll, naturalComparator[T])
 
   def min[T](coll: Collection[_ <: T], comp: Comparator[_ >: T]): T =
-    coll.asScala.min(comp)
+    coll.scalaOps.reduceLeft((a, b) => if (comp.compare(a, b) <= 0) a else b)
```

Comment:
`reduceLeft(math.min)`?

Reply:
That would fail to use the supplied `comp` comparator. (Also `math.min` only works for private numbers, not for any `T`.)